e2e/cypress.json

```diff
@@ -11,5 +11,7 @@
   "ignoreTestFiles": ["**/__snapshots__/*", "**/__image_snapshots__/*"],
   "projectId": "ed7hed",
   "defaultCommandTimeout": 10000,
-  "taskTimeout": 60000
+  "taskTimeout": 60000,
+  "chromeWebSecurity": false,
+  "baseUrl": "http://localhost:3001"
```

Comment:
This line solves a problem I had with e2e tests starting again once all wp plugins were installed, reinstalling them again.

Also, not setting it is an [anti-pattern](https://docs.cypress.io/guides/references/best-practices.html#Setting-a-global-baseUrl).

Reply:
Nice. Should we use `cy.visit("/xxx")` instead of `cy.visit("https://localhost:3001/xxx")` from now on?